lib/binaries.sh

```diff
@@ -115,9 +115,8 @@ install_nodejs() {
   if [ "$code" != "200" ]; then
     echo "Unable to download node: $code" && false
   fi
-  tar xzf /tmp/node.tar.gz -C /tmp
   rm -rf "${dir:?}"/*
-  mv /tmp/node-v"$number"-"$os"-"$cpu"/* "$dir"
```

Comment:
It looks like you are dropping the only usage of `$os` and `$cpu` here. Mind dropping the assignments a few lines up (`os=$(get_os)` and `cpu=$(get_cpu)` )?

Reply:
Done